libs/api/account/data-access/src/lib/api-account-data-access.service.ts

```diff
@@ -84,6 +107,60 @@ export class ApiAccountDataAccessService implements OnModuleInit {
     return solana.getTokenAccounts(accountId, mint.toString())
   }
 
+  async closeAccount(input: CloseAccountRequest): Promise<AppTransaction> {
```

Comment:
CloseAccount and CreateAccount are very similar, I guess we should also include them in the refactor we are planing in #292